app/services/extension.js

```diff
@@ -66,13 +67,27 @@ export default Service.extend({
     set(this, 'extension', extension);
     this.loadSchemaFromShorthand();
     this.syncFragmentsToSchema();
+    this.persist();
   },
 
   setSetting(key, value) {
     set(this, 'data._settings', get(this, 'data._settings') || {});
     set(this, `data._settings.${key}`, value);
   },
 
+  renderPreview(simpleFragments) {
+    if (this.extension._isDummy) {
```

Comment:
why are you checking `_isDummy` here? This means it will only work on local development

Reply:
or is this a dev tool?

Reply:
I was being conservative & leaving it as a dev tool. 
But could totally open it up to production if you think its a good idea!

Reply:
I guess if its in prod, maybe should be collapsible or something

Reply:
mmm yeah lets leave as dev, a lil confusing for non devs

Reply:
I feel like this preview should render the entire payload, rather than just simple fragemnts

Reply:
This also shouldn't attach to `data`, as `data` is the key that actually serializes to Contentful's column